modules/tinymce/src/core/test/ts/atomic/html/Base64UrisTest.ts

```diff
@@ -0,0 +1,134 @@
+import { Assert, UnitTest } from '@ephox/bedrock-client';
+import { extractBase64DataUris, Base64Extract, buildBase64DataUri, restoreDataUris, Base64UriMatch } from 'tinymce/core/html/Base64Uris';
+import { Obj } from '@ephox/katamari';
+import { Logger } from '@ephox/agar';
+
+UnitTest.test('Base64Uris Test', () => {
```

Comment:
This doesn't follow the normal core/plugin test naming convention, does that matter? eg it's normally `atomic.tinymce.core.html.Base64UrisTest`

Reply:
Not sure about the naming convention here I used to have the dot notation but some devs doesn't seem to follow that so everything is inconsistent now. The file name is reported as well as the test name so maybe it's not that important?

Reply:
Yeah, I'm not sure either. Most of the tests in `silver` don't follow this convention, but most other things do so I just thought I'd point it out. I'm not worried either way to be honest 👍